rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2404,23 +2280,57 @@ public String submitPlannings(@HeaderParam("sessionid") String sessionId,
 
     }
 
+    @Override
+    public JobIdData reSubmit(String sessionId, String jobId, PathSegment pathSegment, UriInfo contextInfos)
+            throws JobCreationRestException, NotConnectedRestException, PermissionRestException,
+            SubmissionClosedRestException, IOException {
+        Scheduler scheduler = checkAccess(sessionId, "reSubmit");
+
+        File tmpJobFile = File.createTempFile("job", "d");
+
+        JobId newJobId;
+
+        try {
+            final String jobContent = scheduler.getJobContent(JobIdImpl.makeJobId(jobId));
+            FileUtils.write(tmpJobFile, jobContent, Charset.forName("UTF-8"));
+        } catch (NotConnectedException e) {
+            e.printStackTrace();
+        } catch (UnknownJobException e) {
+            e.printStackTrace();
+        } catch (PermissionException e) {
+            e.printStackTrace();
+        } catch (SubmissionClosedException e) {
+            e.printStackTrace();
+        } catch (JobCreationException e) {
+            e.printStackTrace();
```

Comment:
I'm sure we can do better than printing all stracktraces like this, I found this BTW:
https://stackoverflow.com/questions/7469316/why-is-exception-printstacktrace-considered-bad-practice

Reply:
Yeah. it was mistake from my side. Fixed